index.html

```diff
@@ -165,49 +165,44 @@ <h2 class="visually-hidden">Клиенты</h2>
             </li>
           </ul>
         </section>
-        </div>
+      </div>
   </main>
 
   <footer class="main-footer">
     <section class="contacts">
       <h2 class="visually-hidden">Контакты</h2>
-      <div class="contacts-wrapper">
-        <img class="map" src="images/map.png" width="100%" height="416" alt="Карта проезда">
-        <div class="adress">
+      <div class="contacts-wrapper container">
+        <img class="contacts-map" src="images/map.png" width="1440px" height="416" alt="Карта проезда">
```

Comment:
Атрибуты ширины и высоту должны быть без единиц указаны:
```suggestion
        <img class="contacts-map" src="images/map.png" width="1440" height="416" alt="Карта проезда">
```